torch/fx/passes/splitter_base.py

```diff
@@ -119,6 +120,16 @@ def __init__(
             "we might not care about non-tensor data flow and we can set this option "
             "to true to disable the functionality that prevent non-tensor data flow.",
         )
+        parser.add_argument(
+            "--move-non-tensor-nodes-on-boundary",
+            "--move_non_tensor_nodes_on_boundary",
+            required=False,
+            type=bool,
+            help="AOTI does not support non-tensor nodes on acc->acc, acc->gpu and gpu->acc boundary. "
+            "For non-tensor nodes on acc->acc boundary and acc->gpu, we move the nodes from upstream to downstream. "
```

Comment:
i'm thinking will acc->gpu also already handled by `reduce_acc_nodes_non_tensor_output`

inside `reduce_acc_nodes_non_tensor_output`, we'll find these nodes who are in acc but produce non-tensor output and have users in non-acc, and then move them out of acc